src/Text/Pandoc/Readers/RST.hs

```diff
@@ -505,7 +505,11 @@ includeDirective top fields body = do
          let numberLines = lookup "number-lines" fields
          let classes = trimr lang : ["numberLines" | isJust numberLines] ++
                           maybe [] words (lookup "class" fields)
-         let kvs = maybe [] (\n -> [("startFrom", trimr n)]) numberLines
+         let kvs = maybe [] (\n -> let tn = trimr n
```

Comment:
`null tn` would be better than `tn == ""`. But as a matter of style, I think this is cleaner:
```haskell
case trimr n of
    [] -> []
    xs -> [("startFrom", xs)]
```
Indeed, you could use a list comprehension:
```haskell
let kvs = [("startFrom", xs) | x <- maybeToList (trimr <$> tn)]
```
but this probably isn't as clear to the reader.